### PR TITLE
Updated the wrong links to new correct links in catalog page

### LINKED ISSUE #1600: [Docs] Broken link on catalog page
### Description
The link in the **help-modal** in the **Meshery UI** section is broken.

Steps to reproduce 
- Go to [catalog page](https://meshery.io/catalog)
- Click on **import any catalog item**
- In the Meshery UI section the link in the docs is broken.

In addition to this, in the **Meshery CLI** section the link in `docs` leads to a wrong page.

### Expected Behavior
- Link in the **Meshery UI** should lead to [this page](https://docs.meshery.io/guides/configuration-management/importing-apps#import-apps-using-meshery-ui).
- Link in the **Meshery CLI** should lead to [this page](https://docs.meshery.io/guides/configuration-management/importing-apps#import-apps-using-meshery-cli).

### Screenshot
Docs broken link in modal in catalog

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by ASK-03**
**Description**
This PR fixes #1600 

- Removed wrong link and added correct link in  **Meshery UI & Meshery CLI** modal on catalog page
- When clicked a new page is opened rather than redirecting the user

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by ASK-03** [comment]
Hello, @sudhanshutech, could you please review this pull request? It is identical to [this previous PR](https://github.com/meshery/meshery.io/pull/1605). Unfortunately, the previous PR was closed and could not be recovered.

**Review by vishalvivekm — COMMENTED** [review]
Thanks for fixing the links.

File: _includes/help-modal.html [hunk 1]
```diff
@@ -89,7 +89,7 @@
           <!-- Tab content -->
           <div id="MesheryUI" class="tabcontent">
             <h3>Meshery UI</h3>
-            <p>To import designs, visit <a href="https://docs.meshery.io/guides/meshery-design#import-your-designs">docs</a></p>
+            <p>To import designs, visit <a href="https://docs.meshery.io/guides/configuration-management/importing-apps#import-apps-using-meshery-ui" target="_blank">docs</a></p>
```
Comment (vishalvivekm) [inline _includes/help-modal.html:92]: `target="_blank"`
I'm not sure if we would want these help pages to open in new tab. @sudhanshutech might have a different say here.

**Comment by leecalcote** [comment]
Thanks, @ASK-03 👍

File: _includes/help-modal.html (hunk 1 above)
Comment (sudhanshutech) [reply _includes/help-modal.html:92]: We can open docs in new tab, there are many benefits of it: 
It allows users to maintain their context on the original page. They can refer back to the documentation while still having the original page accessible. In this way we are not forcing user to navigate away from the current page.

**Review by sudhanshutech — APPROVED** [review]

File: _includes/help-modal.html (hunk 1 above)
Comment (sudhanshutech) [inline _includes/help-modal.html:92]: @ASK-03 please change this link to: https://docs.meshery.io/extensions/importing-a-design

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: _includes/help-modal.html (hunk 1 above)
Comment (sudhanshutech) [reply _includes/help-modal.html:92]: Application tab is not used now

Comment (ASK-03) [reply _includes/help-modal.html]: I have done the changes.

**Pull request merged**